docs.openc3.com/plugin.txt

```diff
@@ -0,0 +1,7 @@
+TOOL docs "Documentation"
+  URL /tools/docs/index.html
+  INLINE_URL nil
+  ICON mdi-hexadecimal
```

Comment:
I think this icon should be `mdi-file-document-multiple` but we already use that for handbooks. So for handbooks maybe `view-list-outline` or `list-box` (https://pictogrammers.com/library/mdi/)?

Reply:
changed to mdi-bookshelf